pkg/controllers/nodeclaim/consistency/controller.go

```diff
@@ -101,7 +102,7 @@ func (c *Controller) Reconcile(ctx context.Context, nodeClaim *v1beta1.NodeClaim
 			return reconcile.Result{}, fmt.Errorf("checking node with %T, %w", check, err)
 		}
 		for _, issue := range issues {
-			log.FromContext(ctx).Error(err, "consistency error")
+			log.FromContext(ctx).Error(errors.New(string(issue)), "consistency error")
```

Comment:
I'm guessing we have to change this on main to fit the linting changes too, right?

Reply:
This change is already there on main.